src/lib/utils/strategies/FlagUnorderedStrategy.ts

```diff
@@ -8,13 +8,13 @@ export interface FlagStrategyOptions {
 	 * The accepted flags. Flags are key-only identifiers that can be placed anywhere in the command.
 	 * @default []
 	 */
-	flags?: readonly string[];
+	flags?: readonly string[] | boolean;
 
 	/**
 	 * The accepted options. Options are key-value identifiers that can be placed anywhere in the command.
 	 * @default []
 	 */
-	options?: readonly string[];
+	options?: readonly string[] | boolean;
```

Comment:
Update TSDoc to mention what it means to set `string[]` and what it means to set `boolean`